src/ussd_higherhealth_healthcheck.js

```diff
@@ -224,6 +225,714 @@ go.app = (function () {
+        //prov:[{name, label}]
+        'ZA-EC': [
+            'Buffalo City', 'Eastcape Midlands', 'Ikhala', 'Ingwe', 'King Hintsa', 'King Sabata Dalindyebo (KSD)', 'Lovedale', 'Nelson Mandela University (NMU)', 'Port Elizabeth', 'Rhodes University (RU)', 'UNISA', 'University of Fort Hare (UFH)', 'Walter Sisulu University (WSU)'
+        ],
+        'ZA-FS': [
+            'Central University of Technology (CUT)', 'Flavius Mareka ', 'Goldfields ', 'Maluti ', 'Motheo ', 'UNISA', 'University of the Free State (UFS)'
+        ],
+        'ZA-GT': [
+            'Central Johannesburg', 'Ekurhuleni East', 'Ekurhuleni West', 'North West University (NWU)', 'Sedibeng', 'Sefako Makgatho Health Sciences University (SMU)', 'South West Gauteng ', 'Tshwane North', 'Tshwane South', 'Tshwane University of Technology (TUT)', 'UNISA', 'University of Johannesburg (UJ)', 'University of Pretoria (UP)', 'University of the Witwatersrand (WITS)', 'Vaal University of Technology (VUT)', 'Western'
+        ],
+        'ZA-NL': [
+            'Coastal', 'Durban University of Technology (DUT)', 'Elangeni', 'Esayidi', 'Majuba', 'Mangosuthu University of Technology (MUT)', 'Mnambithi ', 'Mthashana', 'Thekwini', 'UNISA', 'Umfolozi', 'Umgungundlovu ', 'University of Kwazulu Natal (UKZN)', 'University of Zululand (UNIZULU)'
+        ],
+        'ZA-LP': [
+            'Capricorn', 'Lephalale', 'Letaba', 'Mopani', 'Sekhukhune', 'Tshwane University of Technology (TUT)', 'UNISA', 'University of Limpopo (UL)', 'University of Venda (UNIVEN)', 'Vhembe', 'Waterberg'
+        ],
+        'ZA-MP': [
+            'Ehlanzeni', 'Gert Sibande', 'Nkangala', 'Tshwane University of Technology (TUT)', 'UNISA', 'University of Mpumalanga (UMP)', 'Vaal University of Technology (VUT)'
+        ],
+        'ZA-NW': [
+            'North West University (NWU)', 'Orbit', 'Taletso', 'UNISA', 'Vuselela'
+        ],
+        'ZA-NC': [
+            'Northern Cape Rural', 'Northern Cape Urban', 'Sol Plaatje University (SPU)', 'UNISA'
+        ],
+        'ZA-WC': [
+            'Boland', 'Cape Peninsula University of Technology (CPUT)', 'College of Cape Town', 'False Bay', 'Nelson Mandela University (NMU)', 'Northlink', 'South Cape', 'Stellenbosch University (SU)', 'UNISA', 'University of Cape Town (UCT)', 'University of Western Cape (UWC)', 'West Coast']
+      };
+      var selected_prov = self.im.user.answers.state_province;
+      var choices = [];
+      for(var i=0; i<universities_by_province[selected_prov].length; i++){
+        choices[i] = new Choice(
+            universities_by_province[selected_prov][i],
+            $(universities_by_province[selected_prov][i])
+        );
+      }
+      return new PaginatedChoiceState(name, {
+        question: $([
+          "Select your university.",
+          "",
+          "Reply:"
+        ].join("\n")),
+        accept_labels: true,
+        options_per_page: null,
+        choices: choices,
+        next: "state_campus"
```

Comment:
I see we haven't defined the `more` and `back` labels.

I would define them, even if we set them to the default "Back" and "Next" values, so that when the content gets pulled for translation, the "Back" and "Next" also gets pulled and sent to the translators.